x-pack/plugins/uptime/public/pages/settings.tsx

```diff
@@ -148,7 +148,7 @@ export const SettingsPage: React.FC = () => {
   );
 
   return (
-    <EuiPanel style={{ maxWidth: 1000, margin: 'auto' }}>
+    <EuiPanel style={{ maxWidth: 1000, margin: 'auto' }} hasBorder>
```

Comment:
```suggestion
    <EuiPanel hasBorder>
```

I'd remove the maxWidth and centered layout in favor of a left aligned layout without the panel border and padding.

<img width="1474" alt="Screenshot 2021-06-29 at 22 31 39" src="https://user-images.githubusercontent.com/4104278/123863412-dbd52e80-d929-11eb-9993-7cbeeda0b5a2.png">